kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportsConfig.kt

```diff
@@ -672,6 +676,19 @@ public interface KoverReportFilter {
      */
     public fun annotatedBy(vararg annotationName: Provider<String>)
 
+    /**
+     * Add all classes in specified project. Only the project path is used (starts with a colon).
+     *
+     * It is acceptable to use `*` and `?` wildcards,
+     * `*` means any number of arbitrary characters (including no chars), `?` means one arbitrary character.
+     *
+     * Example:
+     * ```
+     *  projects.add(":my:project-path")
```

Comment:
Perhaps you can also add example that shows that wildcards are also working instead of `:`, e.g. `projects.add("*libs")`